package.json

```diff
@@ -40,7 +40,7 @@
     "eslint-plugin-html": "^1.4.0",
     "eslint-plugin-promise": "^1.1.0",
     "eslint-plugin-standard": "^1.3.2",
-    "firebase": "^2.4.1",
+    "firebase": "^3.0.2",
```

Comment:
I think this should be `^2.4.1 || 3.x.x` as that is what we've done with GeoFire and ReactFire. See [these helpers](https://github.com/firebase/reactfire/blob/cb4acfbb2501c1d5262c14ab90a935c88dd308a3/src/reactfire.js#L33-L66) for how to easily make that work.